app.py

```diff
@@ -70,6 +71,29 @@ def viewPatientDetails():
 	return render_template("patients.html",patient_details=patient_details,pageTitle="patients details")
 
 
+## searching for patient deatils using patiendID
+@app.route("/searchPatient")
+def searchPatient(patientId): # takes only one input parameter i.e patientId
+
+        #create dataframe
+        conn_search = sqlite3.connect("hospital.db")
+        df=pd.read_sql_query("SELECT * FROM patients",conn_search)
+        final_df=df[df["ws_pat_id"]==patientId]
+        
+        #render dataframe as html
+        html = final_df.to_html()
+
+        #write html to file
+        text_file = open("search_patient.html", "w")
+        text_file.write(html)
+        text_file.close()
+
+        conn_search.close()
+
+        return render_template("search_patient.html")
```

Comment:
Flask can return HTML files/views only from `templates` directory. so change
```python
open("search_patient.html", "w")
```
to
```python
open("templates/search_patient.html", "w")
```